docs/data.rst

```diff
@@ -109,6 +111,60 @@ all clinical data objects, you always need to specify `patientId`.
 Similar structure is used for pipeline metadata, however, for all pipeline metadata objects,
 you should always include ``sampleId``.
 
++++++++++++++++++++++++++++++++++++++++++
+Specify unique identifiers of the object
++++++++++++++++++++++++++++++++++++++++++
+
```

Comment:
Do you think it should be stated in this section somewhere that if you specify `localId` and all the fields that generate a unique identifier for a record, that the `localId` will be used as the unique identifier? So, if you have an Enrollment record and you specify localId, patientId and enrollmentApprovalDate, localId will be used as the unique identifier.

Reply:
will do